### PR TITLE
Product tests handle interrupt

### PR DESCRIPTION
**Pull request by arhimondr**

### CODE REVIEW

**Comment by arhimondr** [comment]
@kokosing @petroav @fiedukow Please review

File: presto-product-tests/bin/run_on_docker.sh [hunk 1]
```diff
@@ -78,6 +100,14 @@ PRODUCT_TESTS_ROOT="${SCRIPT_DIR}/.."
 PROJECT_ROOT="${PRODUCT_TESTS_ROOT}/.."
 DOCKER_COMPOSE_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker/${ENVIRONMENT}/docker-compose.yml"
 
+SINGLE_NODE_DOCKER_COMPOSE_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker/singlenode/docker-compose.yml"
+DISTRIBUTED_DOCKER_COMPOSE_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker/distributed/docker-compose.yml"
```
Comment (sopel39) [inline presto-product-tests/bin/run_on_docker.sh:104]: for each new test configuration we would have to add line here. Until there are 2 or 3 it is fine, but if there are more, then separate script should be created for each configuration.

**Comment by sopel39** [comment]
lgtm

File: presto-product-tests/bin/run_on_docker.sh (hunk 1 above)
Comment (arhimondr) [inline presto-product-tests/bin/run_on_docker.sh:104]: We don't support running tests on multiple environment in parallel on the same machine. So it is like a `sanity check`.

**Comment by arhimondr** [comment]
merged

File: presto-product-tests/bin/run_on_docker.sh [hunk 2]
```diff
@@ -40,28 +40,50 @@ function stop_unnecessary_hadoop_services() {
   docker exec ${HADOOP_MASTER_CONTAINER} supervisorctl stop zookeeper
```
Comment (kokosing) [inline presto-product-tests/bin/run_on_docker.sh:40]: docker-compose -f conf.yaml hadoop-master run supervisorctl stop zookeeper

**Pull request closed without merging**